tests/annotated/test_annotated_spec.lua

```diff
@@ -4,6 +4,7 @@ local fixer = require("Fixtures.fix_repo")
 
 describe(" Annotated Methods Tests ", function()
     local luagit2 = require("luagit2")
+    local luagit2_annotated = require("annotated.annotated_cover")
```

Comment:
All this wrapping and renaming to get the coverage is not a good design decision. Developers often look into the tests to try to understand how a library is used and what they will find here is not a real usage, but instead this workaround to get the coverage %. I suggest instead to try to get a deeper understanding at how luacov works and see if it is possible to solve this in a different way. Or alternatively, leaving the coverage results out for the moment until a better solution is found.